commands/deploy.go

```diff
@@ -156,7 +167,9 @@ func (c *DeployCmd) Run() (err error) {
 		return fmt.Errorf("failed to trigger app update: %v", err)
 	}
 
-	fmt.Println("Done!")
+	if !c.Quiet{
+		fmt.Println("Done!")
```

Comment:
We should use the `log` interface here so that we don't have to check c.Quiet, it will just be handled automatically by the set log level.